packages/skatejs/src/with-initializer.js

```diff
@@ -0,0 +1,13 @@
+// @flow
+
+export const withInitializer = (Base: Class<any> = HTMLElement): Class<any> =>
+  class extends Base {
+    static get observedAttributes() {
+      let observed = super.observedAttributes;
+      this.initializeCallback();
```

Comment:
I'd like to pull in the comment used in the `observedAttributes` implementation and put it somewhere in here (up to you where it goes).